.github/workflows/build.yml

```
@@ -36,17 +36,10 @@ jobs:
     steps:
       # Note that a duplicate of this step must be added at the top of
       # each job.
-      - uses: GitHubSecurityLab/actions-permissions/monitor@v1
+      - name: Apply standard cisagov job preamble
+        uses: cisagov/action-job-preamble@v1
         with:
-          # Uses the organization variable unless overridden
-          config: ${{ vars.ACTIONS_PERMISSIONS_CONFIG }}
-      # Note that a duplicate of this step must be added at the top of
-      # each job.
-      - id: harden-runner
-        name: Harden the runner
-        uses: step-security/harden-runner@v2
-        with:
-          egress-policy: audit
+          permissions_monitoring_config: ${{ vars.ACTIONS_PERMISSIONS_CONFIG }}
```

I think this still has value to identify the source of this variable.

```suggestion
          # Uses the organization variable unless overridden
          permissions_monitoring_config: ${{ vars.ACTIONS_PERMISSIONS_CONFIG }}
```

---

I think the comment as it stands is very unclear.  How about "Use the cisagov organization variable containing the organization-wide permissions monitoring configuration"?

---

Please see commit 3f1ef79.

---

That's a great start but I don't want to lose the mention of overriding the variable. If you create a variable of the same name in the repository then it overrides the organization variable we are currently leveraging with no changes to workflows required.

---

How about this?

Use a variable to specify the permissions monitoring configuration.  By default this will yield the configuration stored in the cisagov organization-level variable, but if you want to use a different configuration in the case of this particular repository then simply:
* Create a repository-level variable with the same name
* Set this new variable's value to the configuration you want to use for this repository.

Note that there is no need to modify the workflow for the repository variable to take precedence.

---

Please also see commit 2f3814f.